content/en/docs/installation/compatibility.md

```diff
@@ -29,7 +29,7 @@ plane nodes in the [GKE
 docs](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters#add_firewall_rules).
 
 Alternatively, you can read how to [disable the webhook
-component]#Disabling Webhook below. TODO: fix link
+component](./#disabling-webhook).
```

Comment:
I think this is an invalid link as it will go to `index.md`? Change to `(#disabling-webhook)`